src/XIVLauncher.Common/Game/IntegrityCheck.cs

```diff
@@ -108,6 +110,16 @@ private static IntegrityCheckResult DownloadIntegrityCheckForVersion(string game
             {
                 var relativePath = file.FullName.Substring(rootDirectory.Length);
 
+                // for unix compatibility with windows-generated integrity
+                if (PlatformHelpers.GetPlatform() == Platform.Win32 || PlatformHelpers.GetPlatform() ==  Platform.Win32OnLinux)
+                {
+#if DEBUG
+                    Log.Debug($"{relativePath} swapping to {relativePath.Replace("/", "\\")}");
+#endif
+                    relativePath = relativePath.Replace("/", "\\");
```

Comment:
If all path replacements were from `Path.DirectorySeparatorChar` to `"\\"` they could always be made to avoid issues with the platform detection (assuming all paths are supposed to be canonically DOS-style, everything could also be made to use `/` )

Reply:
This is also definitely an option (as opposed to doing a replacement on the string). ...given that the bulk of users are on Windows and past integrity files are built that way, we'd probably be best sticking to backslashes to maintain compatibility (for the rare cases where someone may want to verify an old client version).